dockerfile_parse/parser.py

```diff
@@ -271,6 +271,8 @@ def _clean_comment_line(line):
                 else:
                     current_instruction['content'] += line
                     current_instruction['endline'] = lineno
+
+                    # pylint: disable=unsupported-assignment-operation
                     if current_instruction['value']:
                         current_instruction['value'] += _rstrip_backslash(line)
                     else:
```

Comment:
Careful with pylint disable scope: http://pylint.pycqa.org/en/latest/user_guide/message-control.html
You generally want to disable the warning in the smallest scope possible, here it should be inside the if block

Reply:
I tried to do it, but it _doesn't_ work. If I move disabling comment into if-statement, it will disable the error only for assignments in if-block, not else-block

Reply:
But thanks for the link. I'll be careful with this

Reply:
you can re-enable that check after, if there is no other way:
```python
# pylint: disable=unsupported-assignment-operation
nananana
# pylint: enable=unsupported-assignment-operation
```

Reply:
Oh, the else statement is also an `unsupported-assignment-operation`? I thought it was just the `+=`. Oh well, this works :)